nextjs-task-system/README.md

```
@@ -14,6 +40,11 @@ pnpm dev
 bun dev
 ```
 
+The project has a functional login with role control, use this users to start trying the funcionalities:
+
+- email: user@gmail.com password: user123 role: normal user
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)